index.html

```diff
@@ -0,0 +1,199 @@
+    <link
+      href="https://fonts.googleapis.com/css?family=Lato"
+      rel="stylesheet"
+    />
+    <link
+      rel="icon"
+      type="image/*"
+      href="./assets/images/titleicon.png"
+      alt="Erriecha | Festival"
+    />
+    <link rel="stylesheet" href="./assets/main.css" />
+  <script src="./assets/speakerdocs.js" defer></script>
+    <title>Erriecha Annual | Festival 2023</title>
+  </head>
+  <body>
+    <header>
+      <nav class="navbar-top bg-second">
+        <ul class="navtop-container start-align end-align">
+          <li class="navtop-list">
+            <a class="navtop-link txt-main">
+              <img src="./assets/images/facebook.png" class="navtop-image" />
+            </a>
+          </li>
+          <li class="navtop-list txt-main">
+            <a class="navtop-link">
+              <img src="./assets/images/twitter.png" class="navtop-image" />
+            </a>
+          </li>
+          <li class="navtop-list txt-main">
+            <a class="navtop-link txt-main"> English </a>
+          </li>
+          <li class="navtop-list txt-main">
+            <a class="navtop-link txt-main"> Mypage </a>
+          </li>
+          <li class="navtop-list">
+            <a class="navtop-link txt-main"> Logout </a>
+          </li>
+        </ul>
+      </nav>
+      <nav class="navbar-main bg-white">
+        <ul class="navmain-icon start-align">
+          <li class="navlist-icon listevent-logo">
+            <a href="./index.html" class="listicon-link listicon-image">
+              <img class="linklogo-image" src="./assets/images/titleicon.png" />
+            </a>
+          </li>
+          <li class="navlist-icon listicon-close">
+            <a href="" class="listicon-link txt-second linkicon-close"
+              >&#215;</a
+            >
+          </li>
+         
+        </ul>
+        <ul class="navmain-container end-align">
+          <li class="navmain-list">
+            <a href="./about.html" class="navmain-link txt-third">About</a>
+          </li>
+          <li class="navmain-list">
+            <a href="" class="navmain-link txt-second">Program</a>
+          </li>
+          <li class="navmain-list">
+            <a href="" class="navmain-link txt-second">Join</a>
+          </li>
+          <li class="navmain-list">
+            <a href="" class="navmain-link txt-second">Sponsor</a>
+          </li>
+          <li class="navmain-list list-news">
+            <a href="" class="navmain-link txt-second">News</a>
+          </li>
+          <li class="navmain-list list-campaign">
+            <a href="" class="navmain-link navmain-name txt-third">
+              Erriecha
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <section class="head-section">
+        <div class="headsection-container start-align end-align">
+          <div class="head-humberger mobile-only">
+            <a href="" class="linkicon-humbergur" id="humbergurid"> &#9776;</a>
```

Comment:
- [ ] Your hamburger is not working at this moment. Kindly attend to it and make it work.

[home-hamburger.webm](https://user-images.githubusercontent.com/26597432/226097542-fd644da6-55ee-4584-a144-f283451af273.webm)